FeedKit/images/ImageRepository.swift

```diff
@@ -123,14 +123,24 @@ extension ImageRepository {
 
       return rounded
     }
+    
+    public var identifier: String {
```

Comment:
Did you encounter any issues when migrating to the new `ImageProcessing` API? I was a bit afraid that it was becoming too unwieldy. But this designed seemed to be the only way to implement caching of processed images (addition of the new String identifier) and keep the performance of memory caches (`var hashableIdentifier: AnyHashable`).

Reply:
My intuitive adjustments are causing no obvious issues so far. 

However, the pixels vs points defaults is a bit unsettling. I don’t know if I got that right. Please take extra care documenting this. `ImageProcessor.Resize` describes size in pixels, while initial `units` defaults to `.points`. The relation between `size: CGSize` and `unit: Unit = .points` seems ambiguous.

Reply:
The idea behind this change is to allow users to specify size in points (previously the only option was pixels):

```swift
ImageProcessor.Resize(to: imageView.bounds.size)
```

The `description: String` should be improved, thanks for highlighting this.